carpool/views.py

```diff
@@ -127,6 +128,7 @@ def addPool(request):
             form = PoolForm(request.POST, initial={'paid': False, 'user': request.user})
             if form.is_valid():
                 form.save()
+                messages.success(request,'Ride added successfully')
```

Comment:
I changed the position of `messages.success()`. Originally, it was after the return statement so the control never reached that line.